users/tests/test_views.py

```diff
@@ -188,6 +188,20 @@ def test_user_create_view_should_not_add_new_user_on_post_if_form_is_invalid(sel
         self.assertEqual(response.status_code, 200)
         self.assertEqual(CustomUser.objects.all().count(), 1)
 
+    def test_user_create_view_should_add_new_user_on_post_able_to_login_if_form_is_valid(self):
+        self.client.post(path=self.url, data=self.valid_data)
+
+        new_user = CustomUser.objects.get(email=self.new_user_email)
+        new_user.is_active = True
+        new_user.save()
+
+        response = self.client.post(
+            path="/users/accounts/login/?next=/",
```

Comment:
Please instead of hardcoding string path value use `reverse` django function to call proper endpoint

Reply:
Done